exercises/high-scores/src/test/java/HighScoresTest.java

```diff
@@ -0,0 +1,63 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class HighScoresTest {
+
+    @Test
+    public void shouldDisplayListOfHighScores() {
+        HighScores highScores = new HighScores(new int[] { 30, 50, 20, 70 });
+        assertArrayEquals(new int[] { 30, 50, 20, 70 }, highScores.scores());
+    }
+
+    @Test
+    @Ignore("Remove to run test")
+    public void shouldDisplayLatestAddedScore() {
+        HighScores highScores = new HighScores(new int[]{ 100, 0, 90, 30 });
+        assertEquals(30, highScores.latest());
+    }
+
+    @Test
+    @Ignore("Remove to run test")
+    public void shouldDisplayPersonalBest() {
+        HighScores highScores = new HighScores(new int[] { 40, 100, 70 });
+        assertEquals(100, highScores.personalBest());
+    }
+
+    @Test
+    @Ignore("Remove to run test")
+    public void shouldReturnTopThreePersonalBestFromListOfScores() {
+        HighScores highScores = new HighScores(new int[] { 10, 30, 90, 30, 100, 20, 10, 0, 30, 40, 40, 70, 70 });
+        assertArrayEquals(new Integer[] { 100, 90, 70 }, highScores.personalTopThree());
```

Comment:
You made these `personalTopThree` fit your reference solution (the one in .meta/), but not your implementation stub. The stub expects `int[]`, but your reference solution returns `Integer[]`. To be honest, it really should be `int[]` since the rest of the class uses primitives.